### ISSUE TITLE
Brightness not changing

### ISSUE THREAD

**Issue by epic4gaming8** [sp614x/optifine#5454]
## Description of Issue
Whenever I change the brightness on settings, the brightness never changes, regardless of whatever you changed it or not.

## Steps to Reproduce
Changing the brightness, that's all to it.
## OptiFine Version
The latest version for 1.16.5

## Installation Method
 I put the .JAR file into the mods folder. 

## Other Installed Mods
I have no mods, only resource packs.

## Prior Testing

- Does this happen in Vanilla without OptiFine? No, it only happens in optifine 

## Additional Information
I want optifine to look like vanilla.

**Comment by sp614x** [comment]
Make sure to disable shaders, they can ignore the brightness setting.

**Comment by epic4gaming8** [comment]
It is off, but it's still not working. I reinstalled optifine many times, resetting all settings, but still not working. Brightness is still on moody.

**Comment by sp614x** [comment]
Are shaders set to OFF?
Can you post an F3 screenshot?

**Comment by epic4gaming8** [comment]
Mhm, there off. 
![2021-03-17_22 26 32](https://user-images.githubusercontent.com/80875391/111676769-bfc6da80-87db-11eb-87a6-0cd3f6082d45.png)

**Comment by sp614x** [comment]
Please press F3 and post the debug screenshot.
This is not normal, at night the terrain can't be so bright.
Are you using a resource pack?

**Comment by epic4gaming8** [comment]
No, no resource pack.  
![2021-03-18_12 33 33](https://user-images.githubusercontent.com/80875391/111686609-7c259e00-87e6-11eb-9347-cb77774ed6e4.png)

Pretty sure this is the debug screen

**Comment by sp614x** [comment]
You have enabled FXAA in the shaders menu. 
Can you try with FXAA OFF?

**Comment by epic4gaming8** [comment]
Same thing.

**Comment by sp614x** [comment]
This is very strange, looks like a fullbright mod.
Have you tried "Video Settings -> Other -> Reset" ?
You can open the file `options.txt` in the `.minecraft` folder and check what is `gamma:` set to.

**Comment by epic4gaming8** [comment]
gamma is 0.0. I resetted video settings, but still bright at night.

**Comment by OptiFox** [comment]
I've seen someone with this same exact issue before, the solution for this is to delete both options.txt and optionsof.txt but I don't think that will help sp614x to find the fix for it. So, I'm not that sure

**Comment by epic4gaming8** [comment]
I deleted both files, they're working now. I'll update this if this somehow happens again.

**Comment by epic4gaming8** [comment]
The issue is happening again. The issue was fixed, but when I started optifine again, it happened. I used a resource pack before loading up a world.

**Comment by epic4gaming8** [comment]
I found what's causing the issue. Changing resource packs will cause the issue. It only happens with Optifine.

**Comment by sp614x** [comment]
What resource pack?

**Comment by epic4gaming8** [comment]
Any resource pack will cause it.

**Comment by sp614x** [comment]
> I found what's causing the issue. Changing resource packs will cause the issue. It only happens with Optifine. It will also cause clear water to not work.
There is no option Clear Water in 1.16.5. Can you post a screenshot?

**Comment by epic4gaming8** [comment]
Somehow it's fixed again. The issue's gone now somehow, I don't know what's happening. Changing to any resource pack does cause the issue though,

**Comment by Lentern** [comment]
Can you reproduce this in HD U G8 pre10?

**Comment by epic4gaming8** [comment]
Yes, but with only a certain resource pack. Turns out a X ray pack i was using caused the issue. Unselecting the X ray will fix the issue.

**Comment by NicolasGoeddel** [comment]
I am also using the X-ray resource from time to time and got the same issue. I will try to investigate what the issue could be within the `options.txt` and `optionsof.txt` files.
